docs/index.md

```diff
@@ -5,20 +5,26 @@ permalink: /docs/
 top_graphic: 1
 ---
 
-# General
+# General Subscriber Information
 
-* [ACME Client Software](/docs/client-options/)
-* [Staging Environment](/docs/staging-environment/)
+* [How Let's Encrypt Works](/how-it-works/)
+* [ACME Client Software Options](/docs/client-options/)
```

Comment:
I prefer keeping this title as "ACME Client Software," since "options" can be interpreted as documentation for options like `--help` or `--must-staple`.